quaint/src/ast/column.rs

```diff
@@ -32,6 +32,12 @@ pub struct Column<'a> {
     pub(crate) alias: Option<Cow<'a, str>>,
     pub(crate) default: Option<DefaultValue<'a>>,
     pub(crate) type_family: Option<TypeFamily>,
+    /// Whether the column is an enum.
+    pub(crate) is_enum: bool,
+    /// Whether the column is a (scalar) list.
+    pub(crate) is_list: bool,
```

Comment:
Right, this is needed to cast arrays of enums (?)

Reply:
Yes 👍